openapi.yml

```diff
@@ -149,7 +149,15 @@ components:
       description: Input parameters for the task. Any value is allowed.
 
     Artifact:
-      description: Artifact that the task has produced. Any value is allowed.
+      type: object
+      description: Artifact that the task has produced.
+      properties:
+        filename:
```

Comment:
Now I am thinking, shouldn't this be rather file path? Or should I add optional folder property?